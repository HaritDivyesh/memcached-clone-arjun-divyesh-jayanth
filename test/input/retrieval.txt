1
flush_all
2
set key 1 0 4
test
2
add key 1 0 2
hi
2
replace key 1 0 7
changed
2
append mykey 1 0 3
ing
2
prepend mykey 1 0 3
new
1
get mykey
1
delete key
1
quit
