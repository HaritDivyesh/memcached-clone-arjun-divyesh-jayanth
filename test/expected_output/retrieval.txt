flush_all
OK
set key 1 0 4
test
STORED
add key 1 0 2
hi
NOT_STORED
replace key 1 0 7
changed
STORED
append key 1 0 3
ing
STORED
prepend key 1 0 3
new
STORED
get key
VALUE key 1 13
newchangeding
END
delete key
DELETED
quit
